contrib/docs/new_or_revisit_term_process.md

```diff
@@ -0,0 +1,87 @@
+# Process for New MIxS Term creation or Revising Existing Terms
+
+This document outlines the process for proposing new MIxS terms or revising existing ones.
+
+It is intended to act primarily as a reference guide for the Compliance and Interoperability (CIG) and Technical Working Group (TWG) on the steps for adding a new term, or revisiting an existing term, in the MIxS standard.
+
+## Decision tree
+
+```mermaid
+---
+config:
+  flowchart:
+    subGraphTitleMargin:
+      bottom: 30
+---
+graph TD;
+
+    in_scope{Is the proposed term in scope of MIxS?} -- No --> close_issue_out_of_scope(["Close issue as out of scope"])
+
+
+    in_scope -- Yes --> already_exists{Does the term already exist?}
+
+    already_exists -- "Yes (in MIxS)" --> missing_from_extension["Is the term missing from the target MIxS extension/checklist?"] -- No --> close_issue_already_exists(["Close issue as already exists or out of scope"])
+
+      missing_from_extension -- yes --> change_issue_to_update_extension["Change issue to 'update extension/checklist'"]
+
+   change_issue_to_update_extension --> create_branch
+
+    already_exists -- "Yes (in other standard)" --> reuse_existing
+
+
+    already_exists -- No --> accepted
+
+    subgraph already_exists[Does the term already exist?]
+
+        search_gsc_website("Is the term on the MIxS website (accepted)?")
+        search_github("Is the term on the MIxS Github (in progress)")
+        ask_ai("(Optional) Ask GitHub Copilot AI to identify possible overlaps in other existing MIxS terms")
+        check_other_standards("Search other accepted standards for proposed term")
+    end
+
+    reuse_existing{"Can we import existing term in other standard?"} -- Yes --> accepted
+
+    reuse_existing -- "No (out of scope)" --> close_issue_already_exists
+
+
+
+    accepted(Provisional Accepted) -- Yes --> identify_suitable_checklist_extension("Identify suitable checklist/extension for new term") --> create_branch("Create GitHub branch from issue")
+
+    create_branch --> add_to_yaml("Add new term to source MIxS schema.yaml")
+
+    add_to_yaml --> open_pull_request("Open pull request to MIxS repo")
+
+    open_pull_request --> review("CIG and TWG Review")
```

Comment:
Question: Does this step include the CIG's approval/disapproval following review?